e2e/actors/base.js

```diff
@@ -224,8 +224,6 @@ module.exports = {
 
   async navigateToCaseDetailsAs(user, caseId) {
     await I.goToPage(config.baseUrl);
-    await I.goToPage(config.baseUrl, config.hmctsUser);
```

Comment:
May be replace it with await I.goToPage(config.baseUrl, user);
and comment lines above and below this.